src/manifest.lisp

```diff
@@ -49,6 +62,19 @@ does not exist."
            (return-from manifest-get-by-name matches)))))
 
 
+(defun manifest-update ()
+  "Upgrade the Qi manifest by checking out the latest version from git."
+  (cond ((probe-file (fad:merge-pathnames-as-file +manifest-directory+ "manifest.lisp"))
+         (format t "~%---> Upgrading Qi manifest~%")
+         (run-git-command "fetch origin" +manifest-directory+)
+         (run-git-command "rebase origin/master" +manifest-directory+))
```

Comment:
Yeah, good catch. Since we're abandoning local modifications anyway, would it make sense to just:

```
git reset --hard origin/master
git pull --rebase origin master
```
? Any local changes will surely have a merge conflict might as well get rid of them.